ci/environment.yml

```diff
@@ -1,6 +1,13 @@
 channels:
   - conda-forge
 dependencies:
+########################################################
+# PLEASE READ:
+# When modifying this file, you must also align to match
+# - AB_environments/AB_baseline.conda.yaml
+# - AB_environments/AB_sample.conda.yaml
+########################################################
```

Comment:
Pardon the slightly over-the-top banner, but I've re-done this exact work enough times by now that I think it's deserved.

IMHO writing a CI action that automatically tests alignment would just shift the attrition from myself to the developers.